packages/world/src/NatureLib.sol

```diff
@@ -196,25 +187,26 @@ library NatureLib {
     return result;
   }
 
-  function getRandomOre(Vec3 coord) internal view returns (ObjectTypeId) {
+  function getRandomOre(Vec3 coord) internal view returns (ObjectType) {
     uint256 randomSeed = getRandomSeed(coord);
 
     // Get ore options and their weights (based on remaining amounts)
-    ObjectAmount[] memory oreOptions = new ObjectAmount[](5);
-    uint256[] memory weights = new uint256[](5);
+    ObjectAmount[] memory oreOptions = new ObjectAmount[](6);
+    uint256[] memory weights = new uint256[](6);
 
-    ObjectTypeId[] memory oreTypes = getOreObjectTypes();
+    ObjectType[7] memory oreTypes = ObjectTypeLib.getOreTypes();
     // Use remaining amounts directly as weights
-    for (uint256 i = 0; i < weights.length; i++) {
-      oreOptions[i] = ObjectAmount(oreTypes[i], 1);
-      (, weights[i]) = getCapAndRemaining(oreTypes[i]);
+    // Skip AnyOre (index 0) since it's not a specific ore type
```

Comment:
why include anyOre in `getOreTypes`?

Reply:
it is just the automatic getter generated for all categories

Reply:
if we follow the convention of placing the AnyX type at the beginning of the category we get those cheap checks I wrote